lib/api/src/modules/stories.ts

```diff
@@ -229,7 +230,7 @@ const initStoriesApi = ({
         hierarchyRootSeparator: rootSeparator = undefined,
         hierarchySeparator: groupSeparator = undefined,
         showRoots = undefined,
-      } = (parameters && parameters.options) || {};
```

Comment:
@ndelangen do you think we should come up with a principled way to pull configuration from both parameters and `.getConfig()`? It seems like this comes up a bit and maybe there should be a single API for it?

Reply:
For instance @atanasster does something very similar in https://github.com/storybookjs/storybook/pull/9095